scripts/release_issue_status/update_issue_body.py

```diff
@@ -0,0 +1,97 @@
+import os
+import re
+
+from github import Github
+
+
+def update_issue_body(issue_link):
+    # Get Issue Number
+    issue_number = int(issue_link.replace("https://github.com/Azure/sdk-release-request/issues/", ""))
+    g = Github(os.getenv('TOKEN', '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'))
+    sdk_repo = g.get_repo('Azure/sdk-release-request')
+    rest_repo = g.get_repo('Azure/azure-rest-api-specs')
+    issue_info = sdk_repo.get_issue(number=issue_number)
+    issue_body = issue_info.body
+
+    issue_body_list = [i for i in issue_body.split("\n") if i]
+    # Get the link in issue body
+    for row in issue_body_list:
+        if 'link' in row or 'Link' in row:
+            link = row.split(":", 1)[-1].strip()
+            break
+
+    if link.count('https') > 1:
+        link = link.split(']')[0]
+        link = link.replace('[', "").replace(']', "").replace('(', "").replace(')', "")
+
+    package_name, readme_link = get_pkname_and_readme_link(rest_repo, link)
+    if not package_name and not readme_link:
+        return False
+
+    if not package_name:
+        if 'commit' in readme_link:
+            commit = rest_repo.get_commit()
+            pass
+        readme_link_part = '/specification' + readme_link.split('/specification')[-1]
+        readme_contents = str(rest_repo.get_contents(readme_link_part).decoded_content)
+        package_name = re.findall(r'package-name: (.*?)\\n', readme_contents)[0]
+
+    readme_link = readme_link.replace('python.', '')
```

Comment:
According to modularity principal, these code should be part of `get_pkname_and_readme_link`